docs/guides/volumes/overview/index.md

```diff
@@ -27,27 +27,21 @@ The following diagram shows how Stash takes backup of a stand-alone volume. Open
 
 The backup process consists of the following steps:
 
-1. At first, a user creates a secret with access credentials of the backend where the backed up data will be stored.
-
-2. Then, she creates a `Repository` crd that specifies the backend information along with the secret that holds the credentials to access the backend.
-
-3. Then, she creates a `BackupConfiguration` crd targeting the volume. The `BackupConfiguration` object also specifies the `Task` to use to backup the volume.
-
-4. Stash operator watches for `BackupConfiguration` crd.
-
-5. Once Stash operator finds a `BackupConfiguration` crd, it creates a CronJob with the schedule specified in `BackupConfiguration` object to trigger backup periodically.
-
-6. On next scheduled slot, the CronJob triggers a backup by creating a `BackupSession` crd.
-
-7. Stash operator also watches for `BackupSession` crd.
-
-8. When it finds a `BackupSession` object, it resolves the respective `Task` and `Function` and prepares a backup Job definition.
-
-9. Then, it mounts the targeted volume into the Job and creates it.
-
-10. The Job takes backup of the targeted volume.
-
-11. Finally, when backup is completed, the Job sends Prometheus metrics to the Pushgateway running inside Stash operator pod. It also updates the `BackupSession` and `Repository` status to reflect the backup procedure.
+1. At first, a user creates a `Secret` with access credentials of the backend where the backed-up data will be stored.
+2. Then, she creates a `BackupStorage` custom resource that specifies the backend information, along with the `Secret` containing the credentials needed to access the backend.
+3. KubeStash operator watches for `BackupStorage` custom resources. When it finds a `BackupStorage` object, it initializes the `BackupStorage` by uploading the `metadata.yaml` file into the target storage.
+4. Next, she creates a `BackupConfiguration` custom resource, targeting a standalone volume. The `BackupConfiguration` object specifies the `Repository` pointing to a `BackupStorage` that contains backend information, indicating where to upload backup data. It also defines the `Addon` information with a specified `Task` to be used for backing up the volume.
```

Comment:
It also defines the `Addon` information with a specified `Task` to be used for backing up the volume. -> It also defines the `Addon` information with specified tasks and their configuration parameters to be used for backing up the volume.